### PR TITLE
Add metadata description to pages

### PR DESCRIPTION
**Pull request by igloosi**
Add suitable description meta tag to every page, excluding examples

Iframely: 
screen shot 2018-06-21 at 16 38 57

Trello ticket: https://trello.com/c/bca6gUOv/1048-ensure-metadata-for-pages-in-the-design-system-is-appropriate

### CODE REVIEW

File: src/components/warning-text/index.md.njk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Warning text
-description:
+description: Use the warning text component when you need to warn users about something important, such as legal consequences of an action, or lack of action, that they might
```
Comment (hannalaakso) [inline src/components/warning-text/index.md.njk]: I think there's something missing from this description

File: src/styles/layout/index.md.njk [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 title: Layout
-description:
+description: Organise the layout of the page into blocks
 section: Styles
 aliases:
 backlog_issue_id:
 layout: layout-pane.njk
 ---
 
+Organise the layout of the page into blocks
```
Comment (hannalaakso) [inline src/styles/layout/index.md.njk]: Could this come out for now? Amy said that she'd like to be consulted over any changes to page intros

**Comment by hannalaakso** [comment]
I left a few comments, apart from that looks good 🎉 Facebook sharing hasn't been tested but the iFramely works well.

**Comment by igloosi** [comment]
updated with comments

**Pull request merged**